### DISCUSSION TITLE
Full-screen embedding

### DISCUSSION THREAD

**Discussion by softmarshmallow** [chatwoot/chatwoot#8399] [Feature Requests]
### Is your feature or enhancement related to a problem? Please describe.

While bubble is a good choice for landingpages, We want a solution to use as a full screen (a widget without clicking a bubble button) for our /contact page

### Describe the solution you'd like

Provide a widget that we can embed, without a bubble, unnecessary interactions, dive-in to chat right on.

**Comment by pranavrajs** [comment]
@softmarshmallow Thanks for the feedback. I'm moving this to the feature request discussion, as we will not be prioritizing it in the next releases.

**Comment by tempire** [comment]
Definitely need this. Seems like a small thing, given that you already have the popout window function.
